de.tubs.variantsync.core/src/de/tubs/variantsync/core/monitor/CodeMappingHandler.java

```diff
@@ -32,10 +32,12 @@ public static void addCodeMappingsForDeltas(List<IDelta<?>> deltas) {
 				// Get factory and marker handler for delta
 				IDeltaFactory<?> deltaFactory = DeltaFactoryManager.getFactoryById(delta.getFactoryId());
 				IMarkerHandler markerHandler = deltaFactory.getMarkerHandler();
-				List<IVariantSyncMarker> variantSyncMarkers = markerHandler.getMarkersForDelta(delta.getResource(), delta);
+				List<IVariantSyncMarker> variantSyncMarkers = markerHandler.getMarkersForDelta(delta.getResource(),
+						delta);
```

Comment:
Warum nicht nur `delta` übergeben?

Reply:
This problem will be addressed by the following issue #57